source/Cosmos.System2/Graphics/Image.cs

```diff
@@ -43,6 +44,57 @@ protected Image(uint width, uint height, ColorDepth color)
             Height = height;
             Depth = color;
         }
+
+        /// <summary>
+        /// Resizes the bitmap
+        /// </summary>
+        /// <param name="NewW">the new width of the bitmap</param>
+        /// <param name="NewH">the new height of the bitmap</param>
+        public void Resize(uint NewW, uint NewH)
+        {
+
+            this.RawData = ScaleImage(this, (int)NewW, (int)NewH);
+            this.Width = NewW;
+            this.Height = NewH;
+
+        }
+
+        /// <summary>
+        /// sets a pixel of the image
+        /// </summary>
+        /// <param name="color">the color of the pixel</param>
+        /// <param name="X">the x coordinate of the pixel</param>
+        /// <param name="Y">the y coordinate of the pixel</param>
+        public void SetPixel(Color color,int X,int Y)
+        {
+
+            RawData[X + (Y * Width)] = color.ToArgb();
+
+        }
+
+        static int[] ScaleImage(Image image, int newWidth, int newHeight)
```

Comment:
Missing docstring

Reply:
hi @quajak , this is a private method directly taken from Canvas.cs, in Canvas.cs it didn't have any docstrings

Reply:
We do not want to duplicate the code. Please keep only on version of the function around. I also think we can merge this function with the Resize one